oozie-to-airflow/mappers/fs_mapper.py

```diff
@@ -112,11 +111,18 @@ class FsMapper(ActionMapper):
     Converts a FS Oozie node to an Airflow task.
     """
 
-    tasks: List[Task]
-
-    def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
-        self.tasks = []
+    def __init__(
+        self,
+        oozie_node: Element,
+        name: str,
+        properties: Dict[str, str],
+        trigger_rule: str = TriggerRule.ALL_SUCCESS,
```

Comment:
Why is it necessary? I do not understand this change. The previous version causes fewer conflicts when we change the constructor parameter. I have a plan to change it.